src/services/item/plugins/action/schemas.ts

```diff
@@ -128,7 +130,7 @@ export const getAggregateActions = {
   },
 };
 
-export const exportAction = {
+export const exportAction: FastifySchema = {
```

Comment:
If we want to be able to infer type in the handler we should not type the schema to FastifySchema. You should do `as const satisfies FastifySchema`

This schema will be refactored anyway